metasynthesis/abstract_genetic.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Callable, Tuple
+
+Genome = List[int]
```

Comment:
This might be changed to be simply a list I think. The genome does not necessarily consist of integers.

Reply:
Ah yes good catch!